aws/ec2ids/LICENSE.md

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```

Comment:
Not sure if you need the licence in each subfolder - should be good with just having it at the top level of the repo as it applies to all the code below it.